### ISSUE TITLE
Pyright does not autocomplete parent class' fields

### ISSUE THREAD

**Issue by KaratasFurkan** [microsoft/pyright#962]
Hi, pyright does not autocomplete parent class' fields.

Here is a screenshot with both parent and child class, the point where language server should suggest parent class' field, and expected suggestion.
![Screenshot_20200821_000357](https://user-images.githubusercontent.com/29731420/90825869-3a14a680-e342-11ea-8eaa-5ecd1c627ad7.png)

Just pyright 1.1.64 is enabled as extension, nothing else.

**Comment by erictraut** [comment]
You're correct that parent class fields are not suggested for auto-completion in the context of subclasses. What is the use case for this? Are you overriding a class field and initializing it with a different value than in the base class? I guess that's legal in Python, but it's not a very sound pattern in OOP.

Pyright does autocomplete base class methods (including the full signature) when in the context of a subclass. It's common to override the behavior of a method defined in the base class, so this suggestion makes sense.
![Screen Shot 2020-08-20 at 4 15 02 PM](https://user-images.githubusercontent.com/7040122/90834979-55f75880-e300-11ea-80c5-f82aece8e119.png)

Pyright also offers class variables as suggestions when referencing a parent via a "self" reference.
![Screen Shot 2020-08-20 at 4 11 06 PM](https://user-images.githubusercontent.com/7040122/90834901-25afba00-e300-11ea-8e8e-9cad8c9ac296.png)

**Comment by KaratasFurkan** [comment]
Thanks for quick reply. Overriding class fields is very common in Django. An example:

```python
from django.views.generic import CreateView

class NewPostView(CreateView):
    model = Post
    form_class = PostForm
    success_url = reverse_lazy('post_list')
    template_name = 'new_post.html'
```

**Comment by kadircancetin** [comment]
I agree with @KaratasFurkan . I think overriding class fields used mostly like overriding the constructor method ( `__init__`). It makes code very clean if you need the customize just a few fields.

It is very common on Django views and classes which inherit from views.

https://docs.djangoproject.com/en/3.1/topics/class-based-views/generic-display/
https://docs.djangoproject.com/en/3.1/ref/contrib/admin/
https://www.django-rest-framework.org/tutorial/3-class-based-views/

**Comment by erictraut** [comment]
This is now fixed in Pyright 1.1.65, which I just published. It will also be included in the next version of Pylance.
